src/test/java/net/masterthought/cucumber/ReportBuilderTest.java

```diff
@@ -24,6 +23,7 @@
 import org.junit.Test;
 import org.junit.rules.ExpectedException;
 
+import mockit.Deencapsulation;
```

Comment:
looks like nothing has changed in this file - if so this is good candidate to be reverted